kolibri/core/assets/src/views/loading-spinner/index.vue

```diff
@@ -1,7 +1,9 @@
 <template>
 
-  <div v-show="isVisible" class="loading-spinner-wrapper">
-    <div class="loading-spinner"></div>
+  <div v-if="customStyles" class="loading-spinner"></div>
```

Comment:
I really don't like this. Anyone have better ideas? The element that controls the size was nested, so I couldn't control dimension to match the redline spec. The `translate` styles were also interfering with positioning on the page. @rtibbles

Reply:
How different are the redline specs from the default behaviors? If it's a minor difference, recommend running the defaults by @jtamiace to see whether that is acceptable

Reply:
The default size is way smaller, and the translate will put the positioning off unless we include some spaghetti styles to counteract it. @jtamiace